test/EFCore.PG.FunctionalTests/Query/FullTextSearchDbFunctionsNpgsqlTest.cs

```diff
@@ -271,6 +273,9 @@ SELECT to_tsquery(CAST(@__config_1 AS regconfig), 'a & b')
         [Fact]
         public void WebSearchToTsQuery()
         {
+            if (TestEnvironment.PostgresVersion < new Version(11, 0))
+                return;
```

Comment:
It would be better to skip such tests, and it's possible to do it with xUnit. A simple solution is to write an attribute inherited from `FactAttribute` and override the `Skip` property.
```csharp
public class VersionSpecificAttribute : Attribute
{
    private readonly int _major;
    private readonly int _minor;
    
    public VersionSpecificAttribute(int major, int minor)
    {
    	_major = major;
        _minor = minor;
    }
    
    public Version Version => new Version(_major, _minor);
    
    public override string Skip
    {
    	get => base.Skip ?? (TestEnvironment.PostgresVersion < Version ? "Requires at least PostgreSQL {Version}." : null);
        set => base.Skip = value;
    }
}
```

Reply:
Nice solution, I'll give that a try.